lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -238,7 +334,28 @@ void Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     }
 
     Node *node = nullptr;
-    if (typeName == "MaxPool") {
+
+    if (typeName == "Int8MaxPool" || typeName == "Int8AveragePool") {
```

Comment:
do you need special casing for Int8MaxPool? I think `typeName == "MaxPool"` should handle both cases nicely. Anything i'm missing?

Reply:
Yes. They are the same. However, for "Int8MaxPool", it has the ""Y_zero_point" and "Y_scale". I would like to combine it with Int8AveragePool to check if the fields are there (although for Int8MaxPool, those fields can be ignored).